File: missing_number/missing_number.js
function missing_number(nums) {
  let sum = 0;
  let tot = 0;

  for (let i = 0; i < nums.length; i++) {
    sum += nums[i];
    tot += i + 1;
  }

  return tot - sum;
}

console.log(missing_number([9, 6, 4, 2, 3, 5, 7, 0, 1]));

File: missing_number/missing_number.c
#include <stdio.h>

int missing_number(int* nums, int numsSize) {
  int sum = 0;
  int tot = 0;

  for (int i = 0; i < numsSize; i++) {
    sum += nums[i];
    tot += i + 1;
  }

  return tot - sum;
}

int main(void) {
  int arr[9] = {9, 6, 4, 2, 3, 5, 7, 0, 1};
  printf("%d\n", missing_number(arr, 9));

  return 0;
}